### PR TITLE
Added possibility to add rule overrides to aws managed rules

### PR DESCRIPTION
**Pull request by andreasvikke**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -41,10 +41,12 @@ variable "config" {
...
-      aws_managed_rules = optional(set(string), [
-        "AWSManagedRulesAmazonIpReputationList",
-        "AWSManagedRulesCommonRuleSet",
-      ])
+      aws_managed_rules = optional(map(object({
+        rule_action_override = optional(map(string), {})
+      })), {
+        AWSManagedRulesAmazonIpReputationList = {},
+        AWSManagedRulesCommonRuleSet = {}
+      })
```
Comment (nanohest) [inline variables.tf:49]: what would a valid rule override then look like?
Could you perhaps give an example in the PR description, and preferably add it to README.md and/or eg. `examples/managed_rule_overrides/main.tf`

**Review by nanohest — COMMENTED** [review]
I am missing some documentation

File: variables.tf (hunk 1 above)
Comment (andreasvikke) [reply variables.tf:49]: @nanohest 
I have added some documentation to the README

**Review by nanohest — APPROVED** [review]
LGTM

**Pull request merged**